docs/rtd-pip-requirements

```diff
@@ -1,6 +1,16 @@
-astropy
+### TARDIS requirements ###
+numpy==1.9
+astropy>=1.0.4
+cython>=0.23
+jinja2
+pytest>=2.6.3
+mock
+
```

Comment:
If this section is what the title suggests, some packages are missing in my opinion, such as pandas.